ci/builders/mac_host_engine.json

```diff
@@ -238,20 +238,7 @@
                 "$flutter/osx_sdk": {
                     "sdk_version": "15a240d"
                 }
-            },
-            "tests": [
-                {
-                    "language": "python3",
-                    "name": "Impeller-golden, dart and engine tests for host_release",
-                    "script": "flutter/testing/run_tests.py",
-                    "parameters": [
-                        "--variant",
-                        "ci/host_release",
-                        "--type",
-                        "dart,dart-host,engine"
```

Comment:
That name seems incorrect, it's a copy paste bug since impeller golden is listed in the name, but isn't listed in the `--type` argument.

Reply:
ahh, good catch, these seem like tests we want to run :)

Reply:
@gaaclarke restored this diff, PTAL